ginkgo/generators/generate_command.go

```diff
@@ -28,6 +29,9 @@ func BuildGenerateCommand() command.Command {
 			{Name: "template", KeyPath: "CustomTemplate",
 				UsageArgument: "template-file",
 				Usage:         "If specified, generate will use the contents of the file passed as the test file template"},
+			{Name: "templatedata", KeyPath: "CustomTemplateData",
```

Comment:
could we make this `template-data` to be more consistent with the rest of Ginkgo's multi-word flags?  Thanks!

Reply:
ack, will do